src/panels/config/users/ha-config-users.ts

```diff
@@ -43,36 +43,73 @@ export class HaConfigUsers extends LitElement {
           ),
           sortable: true,
           filterable: true,
+          width: "25%",
           direction: "asc",
           grows: true,
           template: (name) => html`
             ${name ||
             this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
           `,
         },
+        username: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.username"
+          ),
+          sortable: true,
+          filterable: true,
+          width: "20%",
+          direction: "asc",
+          grows: true,
+          template: (username) => html`
+            ${username ||
+            this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
+          `,
+        },
         group_ids: {
           title: this.hass.localize(
             "ui.panel.config.users.picker.headers.group"
           ),
           sortable: true,
           filterable: true,
-          width: "30%",
+          width: "20%",
           template: (groupIds) => html`
             ${this.hass.localize(`groups.${groupIds[0]}`)}
           `,
         },
+        is_active: {
```

Comment:
Is it worth showing this? We don't allow the user to change it.

Reply:
I was thinking about adding that option in the future, so that owners can for example enable and disable guest users when needed.

Reply:
Then we should show it in the UI when we can do that.

Reply:
I added it now. Took a lot longer than hoped, but that was my own unfamiliarity with some backend aspects.
https://github.com/home-assistant/core/pull/43463 and https://github.com/home-assistant/frontend/pull/7757

Should those be merged first and then we continue with this PR?

Reply:
Let's do that in a new PR